LibertyDSNP/example-client
# Feature/build profile component/177348143

# Problem

As a user, I want to view my profile information and others' profile information.
[link to Pivotal Tracker #177348143](https://www.pivotaltracker.com/story/show/177348143)

# Solution

follow the wire frames to create the component and styling that displays your profile information

## Change summary:

- created `userAvatar` component to have a reusable avatar component
- created component that houses profile elements
- created component that builds list of following/follows me list
- created temp data structure with relationship info
- added styling

## Steps to Verify:

1. make sure code structure looks good
2. make sure it looks like the wire frame

## Screenshots (optional):
